.vscode/launch.json

```diff
@@ -19,16 +19,7 @@
 			"request": "launch",
 			"name": "Launch Tests",
 			"program": "${workspaceRoot}/node_modules/mocha/bin/_mocha",
-			"args": [
-				"-u",
-				"tdd",
-				"--timeout",
-				"999999",
-				"--colors",
-				"-r",
-				"ts-node/register",
-				"${workspaceRoot}/test/*.test.ts"
-			],
+			"args": ["-u", "tdd", "--timeout", "999999", "--colors", "-r", "ts-node/register", "${workspaceRoot}/test/*.test.ts"],
```

Comment:
This looks like an odd "fix" for prettier. Usually it expands into multiple lines.

Reply:
It depends on `"printWidth": 130`

Reply:
ah, so the wider print width allows it to not expand

Reply:
Yes